### PR TITLE
I created the /POST office endpoint

### PR DESCRIPTION
**Pull request by chiboycalix**
#### What does this PR do?
          I created on of the /POST office endpoint
#### Description of Task to be completed?
         I created the endpoint for /api/v1/offices 
         I ensured it is working
#### How should this be manually tested?
        It can be tested manually using postman
#### What are the relevant pivotal tracker stories?
       As an Admin I should be able to create a political office-163545004

### CODE REVIEW

File: server/test/officeTest.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      .post('/api/v1/offices')
+      .send(correctDetails)
+      .end((request, response) => {
+        response.body.statusMessage.should.equal('Office created successfully');
+        response.should.have.status(201);
+        done();
+      });
+  });
+  
+  it('should not post an Office with a missing field', (done) => {
```
Comment (tutugodfrey) [inline server/test/officeTest.js]: @chiboycalix great job. Best practice for writing tests requires that you test for edge cases first before testing when correct data is supply. It would make sense if the test for incomplete detail come before the test for complete details. How about that?

File: server/test/officeTest.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    name: 'party name',
+  };
+  
+  it('it should be able to POST a party with all fields being entered', (done) => {
+    chai.request(server)
+      .post('/api/v1/offices')
+      .send(correctDetails)
+      .end((request, response) => {
+        response.body.statusMessage.should.equal('Office created successfully');
+        response.should.have.status(201);
```
Comment (tutugodfrey) [inline server/test/officeTest.js]: Apart from the status code, you can also check whether some of the data returned represent what you are expecting.

File: server/routes/officeRoutes.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import OfficeController from '../controllers/OfficeController';
```
Comment (tutugodfrey) [inline server/routes/officeRoutes.js]: What did you export and what are you importing

File: server/test/officeTest.js (hunk 1 above)
Comment (chiboycalix) [reply server/test/officeTest.js]: Okay, I never thought of that, I'll do that going forward. It's actually intuitive to do it this way. Thank you for point this out to me.

File: server/test/officeTest.js (hunk 2 above)
Comment (chiboycalix) [reply server/test/officeTest.js]: Ok I will implement that hence forth

File: server/routes/officeRoutes.js (hunk 3 above)
Comment (chiboycalix) [reply server/routes/officeRoutes.js]: I exported the router module from "const router = express.Router()",
I imported OfficeController from my controller file so that i can have access to the function createOffice i created there

Comment (tutugodfrey) [reply server/routes/officeRoutes.js]: Check the casing of the OfficeController from where you import it. Since it is an instance of the class, the casing should be camelCase.

Comment (chiboycalix) [reply server/routes/officeRoutes.js]: ok let me do that immediately
